scripts/ksqlDB/statements.sql

```diff
@@ -1,4 +1,4 @@
-CREATE STREAM wikipedia WITH (kafka_topic='wikipedia.parsed', value_format='AVRO');
+CREATE STREAM wikipedia WITH (kafka_topic='wikipedia.parsed', format='AVRO');
```

Comment:
Does `wikipedia.parsed` actually have an Avro key? That's what this stream definition is assuming.

Reply:
Nope.  Per https://github.com/confluentinc/cp-demo/pull/368#issuecomment-827946733, the keys in the messages for `wikipedia.parsed` topic are `null` so there is no schema for `wikipedia.parsed-key` in Schema Registry (it legitimately does not exist).